contracts/CO30_Rob_Byelomorye.IslaPera/edit_me/briefing/situation.html

```diff
@@ -0,0 +1,29 @@
+<font color='#e3d310'>EMPLOYER</font>
+Name: Columbian Government
+Location: Nation Wide
+
+Given the untimely deadline given to it by the rebels, and issues in supplying it's own forces, the federal government has reached out to Synixe to assist in revolving this matter.
```

Comment:
```suggestion
Given the untimely deadline given to it by the rebels, and issues in supplying it's own forces, the national government has reached out to Synixe to assist in revolving this matter.
```

Nitpicky but Columbia is not a federal country